src/libpgmoneta/json.c

```diff
@@ -738,6 +743,11 @@ fill_value(char* str, char* key, uint64_t* index, struct json* o)
       idx++;
       while (idx < len && str[idx] != '"')
       {
+         // check for escape characters in value
+         if(handle_escape_char(str, &idx, len))
```

Comment:
Does the logic still work when you are dealing with `\t` or `\n`?

Perhaps you should still do append inside and only go into the function when you actually find the `\`.

Reply:
> Does the logic still work when you are dealing with `\t` or `\n`?
> 

Actually I was not thinking of implementing for `\n` and `\t` at all.

> Perhaps you should still do append inside and only go into the function when you actually find the `\`.

In that case too when we encounter `\n` or `\t` what will we append to the key or value as a character? This is where I was stuck and thought to not provide support for `\n` and `\t`. Like with what character we will replace `\n` and `\t`.

Reply:
Append `'\n'` or `'\t'` of course~ It's still one character.

Reply:
Just note that in to_string you don't have to do any conversion, since printf() recognizes them and takes care of it.